sycl/test-e2e/format.py

```diff
@@ -389,12 +389,22 @@ def map_result(features, code):
                 elif code is lit.Test.FAIL:
                     code = lit.Test.XFAIL
             return code
+        def check_leak(output):
+            keyword_found = False
+            for line in output.splitlines():
+                if keyword_found and "LEAK" in line:
+                    return lit.Test.FAIL
+                if "Check balance of create/destroy calls" in line:
+                    keyword_found = True
+            return lit.Test.PASS
 
         if len(triples) == 1 and test.config.test_mode == "build-only":
             result.code = map_result(test.config.available_features, result.code)
         if len(devices_for_test) == 1:
             device = devices_for_test[0]
             result.code = map_result(test.config.sycl_dev_features[device], result.code)
+        if test.config.ur_l0_leaks_debug and result.code is lit.Test.PASS:
```

Comment:
I get: `AttributeError: 'TestingConfig' object has no attribute 'ur_l0_leaks_debug'` when ur_l0_leaks_debug is not set. Perhaps you need to do something like this: `litConfig.params.get("ur_l0_leaks_debug")`